atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -50,34 +48,26 @@ def run(self):
             self.log.debug("release set explicitly so not incrementing")
             return
 
-        # No release labels are set so set them.
         component_label = get_preferred_label_key(dockerfile_labels,
                                                   'com.redhat.component')
         try:
             component = dockerfile_labels[component_label]
         except KeyError:
             raise RuntimeError("missing label: {}".format(component_label))
 
-        target_info = self.xmlrpc.getBuildTarget(self.target)
```

Comment:
I don't think that logic belongs in this plugin. From this plugin's point of view a target is not required at all.